packages/metrics/src/features/registration/pointGenerator.ts

```diff
@@ -117,11 +117,19 @@ export const generateInCompleteFieldPoints = async (
       return {
         measurement: 'in_complete_fields',
         tags,
-        fields
+        fields,
+        timestamp: toInfluxTimestamp(task.lastModified)
```

Comment:
I had to also add explicit timestamps for data we write into Influxdb. 
Please read carefully through these timestamp creations. Am I using the right timestamp for each use case?